doc/getting_started/beginner_tutorial.rst

```diff
@@ -25,163 +22,245 @@ Create a Repo
+
+.. code-block:: shell
 
-Unlike Git though, commits in Pachyderm must be explicitly started and finished as they can contain huge amounts of data and we don't want that much "dirty" data hanging around in an unpersisted state. The ``-c`` flag we used above specifies that we want to start a new commit, add data, and finish the commit in a convenient one-liner.
+# If we list the repos, we can see that there is now data
+$ pachctl list-repo
+NAME                CREATED             SIZE
+images              5 minutes ago   57.27 KiB
+
+# We can view the commit we just created
+$ pachctl list-commit images
+REPO                ID                                 PARENT              STARTED            DURATION            SIZE
+images              7162f5301e494ec8820012576476326c   <none>              2 minutes ago      38 seconds          57.27 KiB
+# And view the file in that commit
+$ pachctl list-file images master
+NAME                TYPE                SIZE
+liberty.png         file                57.27 KiB
+ ...
 
-Finally, we can see the data we just added to Pachyderm.
+Finally, we can view the file we just added to Pachyderm. Since this is an image, we can't just print it out in the terminal, but the following commands will let you view it easily.
 
 .. code-block:: shell
+ 
+# on OSX
+$ pachctl get-file images master liberty.png | open -f -a /Applications/Preview.app
 
- # If we list the repos, we can see that there is now data
- $ pachctl list-repo
- NAME                CREATED             SIZE
- data                12 minutes ago       874 B
-
- # We can view the commit we just created
- pachctl list-commit data
- BRANCH              REPO/ID         PARENT              STARTED             FINISHED            SIZE
- master              data/master/0   <none>              6 minutes ago       6 minutes ago       874 B
-
- # We can also view the contents of the file that we just added
- $ pachctl get-file data master sales
- orange 	4
- banana 	2
- banana 	9
- orange 	9
+# on Linux
+ $ pachctl get-file images master liberty.png | display
  ...
 
 Create a Pipeline
 ^^^^^^^^^^^^^^^^^
 
-Now that we've got some data in our repo, it's time to do something with it.
-``Pipelines`` are the core primitive for Pachyderm's processing system (pps) and
-they're specified with a JSON encoding. For this example, we've already created the pipeline for you and it can be found at `examples/fruit_stand/pipeline.json on Github <https://github.com/pachyderm/pachyderm/blob/master/doc/examples/fruit_stand/pipeline.json>`_. Please open a new tab to view the pipeline while we talk through it.
+Now that we've got some data in our repo, it's time to do something with it. ``Pipelines`` are the core primitive for Pachyderm's processing system (pps) and they're specified with a JSON encoding. For this example, we've already created the pipeline for you and you can find the `code on Github <https://github.com/pachyderm/pachyderm/blob/master/doc/examples/opencv>`_. 
```

Comment:
Again, just a question of whether we want to use PFS/PPS language or just Pachyderm pipeline.  I've been trying to about PFS/PPS.